core/src/test/resources/sql/functions.iq

```diff
@@ -549,4 +549,47 @@ limit 3;
 
 !ok
 
+# -----------------------------------------------------------------------------
+# [CALCITE-5745] CONCAT2 function(enabled in Oracle library) gets wrong result when one of the argument is NULL
+!use oraclefunc
+select concat('a', 'b');
++--------+
+| EXPR$0 |
++--------+
+| ab     |
++--------+
+(1 row)
+
+!ok
+
+select concat('a', null);
++--------+
+| EXPR$0 |
++--------+
+| a      |
++--------+
+(1 row)
+
+!ok
+
+select concat('a', cast(null as varchar));
```

Comment:
Nitpick: for the sake of completeness, I think we should add also the test scenarios of `null` as first parameter, and `non-null` as second.